### PR TITLE
ISDB-267 Changes

### PR DESCRIPTION
**Pull request by adetolacog**
**Before creating a pull request make sure that:**

- [ ] commit messages are meaningful and follow good commit message guidelines
- [ ] README and other documentation has been updated / added (if needed)
- [ ] tests have been updated / new tests has been added (if needed)

Please remove this line and everything above and fill the following sections:

### Change description ###

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[ ] No
```

### CODE REVIEW

File: src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      "name":"applicant2_address",
+      "value":"123 test street, London"
+    },
+    {
+      "name": "applicant2_relationToChild",
+      "value": "Step Mother"
+    }
+  ],
+  "form_type": "A60"
+}
```
Comment (hmcts-kodatala) [inline src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json:45]: need to add scan document objects in json

File: src/functionalTest/resources/responses/bulk-scan-a60-transform-output.json [hunk 2]
```diff
@@ -0,0 +1,30 @@
...
+            "applicant2Address":"123 test street, London",
+            "applicant2TelephoneNumber":"09999999990",
+            "applicant2RelationToChild":"Step Parent"
+          }
+        }
+      ]
+    }
+  },
+  "warnings":[
+  ]
```
Comment (hmcts-kodatala) [inline src/functionalTest/resources/responses/bulk-scan-a60-transform-output.json:29]: need to add scan document objects in json

File: src/functionalTest/resources/responses/bulk-scan-a60-validation-output.json [hunk 3]
```diff
@@ -0,0 +1,13 @@
...
+    "items":[
+
+    ]
+  },
+  "errors":{
+    "items":[
+      "applicant1_telephoneNumber is not valid phone number"
+    ]
+  }
+}
```
Comment (hmcts-kodatala) [inline src/functionalTest/resources/responses/bulk-scan-a60-validation-output.json:13]: there is no success response json

File: src/test/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanA60ServiceTest.java [hunk 4]
```diff
@@ -12,22 +14,35 @@
...
 class BulkScanA60ServiceTest {
+    private final ObjectMapper mapper = new ObjectMapper();
+    private static final String APPLICANT1_TEST_ADDRESS = "123 test street, London";
+    private static final String BULK_SCAN_APPLICANT_DETAILS_FIELD = "applicant";
+    private static final String APPLICANT1_TEST_FIRSTNAME = "applicant1_firstName";
+    private static final String APPLICANT1_TEST_LASTNAME = "applicant1_lastName";
+    private static final String A60_TRANSFORM_RESPONSE_PATH
+        = "classpath:response/bulk-scan-a60-transform-output.json";
 
     @Spy
```
Comment (hmcts-kodatala) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanA60ServiceTest.java:47]: why both spy and auto-wired????

File: src/test/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanA60ServiceTest.java [hunk 5]
```diff
@@ -47,42 +62,76 @@ void testA60MandatoryErrorWhileDoingValidation() {
...
+            bulkScanValidationService
+                .transform(BulkScanTransformationRequest
+                               .builder()
+                               .ocrdatafields(TestDataUtil.getA60Data()).build());
+
+        JSONAssert.assertEquals(
+            readFileFrom(A60_TRANSFORM_RESPONSE_PATH),
+            mapper.writeValueAsString(bulkScanTransformationResponse), true
+        );
+    }
```
Comment (hmcts-kodatala) [inline src/test/java/uk/gov/hmcts/reform/bulkscan/services/BulkScanA60ServiceTest.java:136]: need to add test case java documentations

File: src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json [hunk 6]
```diff
@@ -0,0 +1,45 @@
...
+      "value":"09999999990"
+    },
+    {
+      "name":"applicant1_address",
+      "value":"123 test street, London"
+    },
+    {
+      "name": "applicant1_relationToChild",
+      "value": "Step Father"
+    },
```
Comment (hmcts-kodatala) [inline src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json:22]: pls check below tag is required:
 {
      "name": "applicant1_relationToChild",
      "value": "Step Father"
    },

File: src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json [hunk 7]
```diff
@@ -0,0 +1,45 @@
...
+      "name":"applicant2_telephoneNumber",
+      "value":"09999999990"
+    },
+    {
+      "name":"applicant2_address",
+      "value":"123 test street, London"
+    },
+    {
+      "name": "applicant2_relationToChild",
+      "value": "Step Mother"
```
Comment (hmcts-kodatala) [inline src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json:41]: {
      "name": "applicant2_relationToChild",
      "value": "Step Father"
    }, pls this , which not required for ur tcase type

**Review by hmcts-kodatala — COMMENTED** [review]
pls check all review comments

File: src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json (hunk 7 above)
Comment (adetolacog) [reply src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json:41]: This added in order to test optional fields for A60

File: src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json (hunk 6 above)
Comment (adetolacog) [reply src/functionalTest/resources/requests/bulk-scan-a60-transform-input.json:22]: Testing optional field on A60

**Review by hmcts-kodatala — APPROVED** [review]
Ok

**Review by rahulsanjava19 — APPROVED** [review]

**Pull request merged**